libs/contact/tsconfig.json

```diff
@@ -2,30 +2,27 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "paths": {
-      "@daffodil/core": [
-        "dist/core"
+      "@daffodil/*": [
+        "dist/*"
       ],
-      "@daffodil/core/testing": [
-        "dist/core/testing"
-      ],
-      "@daffodil/core/graphql": [
-        "dist/core/graphql"
-      ],
-      "@daffodil/core/state": [
-        "dist/core/state"
+      "@daffodil/contact": [
+        "libs/contact/src"
       ],
-      "@daffodil/driver": [
-        "dist/driver"
+      "@daffodil/contact/driver": [
+        "libs/contact/driver/src"
       ],
-      "@daffodil/driver/hubspot": [
-        "dist/driver/hubspot"
+      "@daffodil/contact/driver/in-memory": [
+        "libs/contact/driver/in-memory/src"
       ],
-      "@daffodil/contact": [
-        "libs/contact/src"
+      "@daffodil/contact/driver/hubspot": [
+        "libs/contact/driver/hubspot/src"
       ],
       "@daffodil/contact/testing": [
```

Comment:
This package name is wrong. It also looks like you're missing state/testing?

Reply:
There's also a directory that needs deleted: `contact/testing/**`

Reply:
Okay both fixed in 6deedfcb77d603ec3c6183bb04c499ef8a539067.